api/config/packages/gesdinet_jwt_refresh_token.yaml

```diff
@@ -0,0 +1,11 @@
+gesdinet_jwt_refresh_token:
```

Comment:
The default of this package for the refresh token is 1 month.
The article of google said 24h which seemed a little strict for an application you don't use every day. https://cloud.google.com/apigee/docs/api-platform/antipatterns/oauth-long-expiration
I did not find other recommendations.
(But i read somewhere that there are applications which even use 90 days)

What do you say @carlobeltrame?

Reply:
We start by not reducing it and making it configurable.